invenio_communities/assets/semantic-ui/less/invenio_communities/theme.less

```diff
@@ -154,3 +157,17 @@
     margin-top: 0;
   }
 }
+
+.community-description-button{
+  background: none!important;
+  border: none;
+  color: @link-color;
+  cursor: pointer;
```

Comment:
I think we have the same problem as we hit once in ILS. 
Why not use the `<a>` element which exists already instead of redefining a new CSS class which is practically the same?

Reply:
why do we still need a pointer rule?
if I remember correctly we just need adding href="#/" attribute to the <a tag to have the cursor
same for the change below